python/hawkey/subject-py.cpp

```diff
@@ -276,40 +278,47 @@ get_best_parser(_SubjectObject *self, PyObject *args, PyObject *kwds, HyNevra *n
     PyObject *with_provides = NULL;
     PyObject *with_filenames = NULL;
     PyObject *with_src = NULL;
+    PyObject *exclude_flags_obj = NULL;
     const char *kwlist[] = {"sack", "with_nevra", "with_provides", "with_filenames", "forms",
-        "with_src", NULL};
+        "with_src", "exclude_flags", NULL};
 
-    if (!PyArg_ParseTupleAndKeywords(args, kwds, "O!|O!O!O!OO!", (char**) kwlist, &sack_Type, &sack,
+    if (!PyArg_ParseTupleAndKeywords(args, kwds, "O!|O!O!O!OO!O", (char**) kwlist, &sack_Type, &sack,
         &PyBool_Type, &with_nevra, &PyBool_Type, &with_provides,
-        &PyBool_Type, &with_filenames, &forms, &PyBool_Type, &with_src)) {
-        return NULL;
+        &PyBool_Type, &with_filenames, &forms, &PyBool_Type, &with_src, &exclude_flags_obj)) {
+        return std::make_pair(nullptr, std::unique_ptr<libdnf::Nevra>());
```

Comment:
Please return NULL. NULL is required to raise an exception when parsing argument fails.